curriculum/test/test-challenges.js

```diff
@@ -88,7 +88,7 @@ const handleRejection = err => {
   }
 };
 
-const dom = new jsdom.JSDOM('');
+const dom = new jsdom.JSDOM('', { resources: 'usable' });
```

Comment:
Did this fix the issue, or was the overflow really caused by the tests searching the wrong element?

Reply:
Nothing has changed since November, so I think it fixed itself.  I'm going to revert all the 'fixes' see if it's still fine.